README.md

```diff
@@ -59,14 +59,20 @@ npm install palindrom --save
 Then you can require it CommonJS or ES6/TS style:
 
 ```js
-//CommonJS
-var Palindrom = require('palindrom');
+// CommonJS
+const Palindrom = require('palindrom').Palindrom;
+const PalindromDOM = require('palindrom').PalindromDOM;
 
 // ES6/TS
-import Palindrom from 'palindrom'
+import { Palindrom } from 'palindrom';
+import { PalindromDOM } from 'palindrom';
 ```
 
-###### Or just download it manually from [github](https://github.com/Palindrom/Palindrom/archive/master.zip).
+Note: The NPM package uses an entry point without a default export. Depending on your environment (Node or browser), you should choose one of the two named exports available: `Palindrom` and `PalindromDOM`.
```

Comment:
I'd say "you could" as I see reasonable use cases to use both in browser.

Reply:
I deleted this whole sentence :)